config/default/kustomization.yaml-e

```diff
@@ -0,0 +1,74 @@
+# Adds namespace to all resources.
```

Comment:
Is this file supposed to be included in this for or is `kustomization.yaml-e` a typo?

Reply:
Removed the file, it should not be included